catroid/gradle/intellij_config_tasks.gradle

```diff
@@ -29,6 +29,7 @@ task gitSkipWorktreeForIntellijConfigFiles << {
         'git update-index --skip-worktree Catroid.iml'.execute().text.trim()
         'git update-index --skip-worktree catroid/src/test/catroidSourceTest.iml'.execute().text.trim()
         'git update-index --skip-worktree .idea/codeStyleSettings.xml'.execute().text.trim()
+        'git update-index --skip-worktree catroid/catroid.iml'.execute().text.trim()
```

Comment:
I looked into it and it seems that gradle is based on Groovy or at least accepts Groovy style. So I wrote a list that can be looped through. I can do [CAT-2258](https://jira.catrob.at/browse/CAT-2258), but need access rights in Jira. 
For now I will close the PR and reopen it when I clear the access issue.